README.md

```diff
@@ -400,9 +358,6 @@ By adding any files above you will get reporting on those files as part of your
 
 By default Coverband has assumed you are trying to track your application code usage not all the supporting framework and library (Gems) code usage. There has been some good points and reasons folks want to track library usage, for example to find out which Gems they aren't actually using in production. See some of the discussion on [issue 21](https://github.com/danmayer/coverband/issues/21).
 
-* Using the Tracepoint Collector
-   * just make sure to set `Coverband.configuration.include_gems = true` in your config options
-   * note by tracking all the additional usage this can have significant performance impacts of the Tracepoint implementation
 * Using the Coverage Collector
```

Comment:
Since there is only 1 kind of collector now, we might want to reword this to be more informational. Like, "How it works" or something like that. Perhaps also moving to a wiki page?

Reply:
updated and cleaned up a bit. Yeah, I do think some of these might be good to move to the Wiki page. I think I will do that outside of this PR as it is getting a bit big ;)